domain-model/src/main/java/uk/gov/hmcts/cmc/domain/orders/HearingCourtType.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.cmc.domain.orders;
+
+public enum HearingCourtType {
```

Comment:
We might have a problem here, @sabahirfan Introduced the class `PilotCourt` and we might have to get rid of either this one, or make those work together

Reply:
Agree this can be merged with `uk.gov.hmcts.cmc.domain.models.directionsquestionnaire.PilotCourt`